packages/react-component-library/src/enhancers/withFormik.tsx

```diff
@@ -20,19 +20,16 @@ const withFormik = (FormComponent: React.FC<any>) => ({
   const formComponentClassNames = classNames(className, {
     'is-invalid': hasError,
   })
+  const errorMessage = hasError ? errors[field.name] : undefined
```

Comment:
Some Lodash `get` could potentially tidy this up. Have we made a decision not to use Lodash?

Reply:
Not at all, lodash is a good idea.